res/skins/Shade/mixer_panel.xml

```diff
@@ -558,6 +558,63 @@
                 </Connection>
               </PushButton>
 
+              <!--
+              **********************************************
+              Button- Toggle Recording / Visual- Recording State
+              **********************************************
+              -->
+              <!--Button as visual indicator for Recording State-->
+              <PushButton>
+              <TooltipId>toggle_recording</TooltipId>
+                <NumberStates>3</NumberStates>
+                <State>
+                  <Number>0</Number>
+                  <Pressed>skin:/btn/btn_record.png</Pressed>
+                  <Unpressed>skin:/btn/btn_record.png</Unpressed>
+                </State>
+                  <State>
+                  <Number>1</Number>
+                  <Pressed>skin:/btn/btn_record_overdown.png</Pressed>
+                  <Unpressed>skin:/btn/btn_record_overdown.png</Unpressed>
+                </State>
+                <State>
+                  <Number>2</Number>
+                  <Pressed>skin:/btn/btn_record_over.png</Pressed>
+                  <Unpressed>skin:/btn/btn_record_over.png</Unpressed>
+                </State>
+                <Pos>214,18</Pos>
+                <Connection>
+                  <ConfigKey>[Recording],toggle_recording</ConfigKey>
+                  <EmitOnDownPress>true</EmitOnDownPress>
+                  <ButtonState>LeftButton</ButtonState>
+                </Connection>
+                <Connection>
+                  <ConfigKey>[Recording],status</ConfigKey>
+                </Connection>
+              </PushButton>
+
+               <!--
+              **********************************************
+              Text- Recording Duration
+              **********************************************
+              -->
+              <RecordingDuration>
+                <Style>
+                  QLabel {
+                    font-size: 8px;
+                    background-color: transparent;
+                    color: #191F24;
+                    text-align:center;
+                  }
+                </Style>
+                <ObjectName>RecDuration</ObjectName>
+                <TooltipId>recording_duration</TooltipId>
+                <Pos>212,8</Pos>
+                <Size>32f,8f</Size>
+                <InactiveText>--:--</InactiveText>
+                <Alignment>center</Alignment>
+               </RecordingDuration>
```

Comment:
this is too dark, and too small -- even for Shade ;)

`bold` and `9px` give better readability and contrast are better in all themes:
![image](https://user-images.githubusercontent.com/5934199/119739146-0ee75680-be82-11eb-869d-9584b11b8c25.png)
And longer durations still fit:
![image](https://user-images.githubusercontent.com/5934199/119738773-849ef280-be81-11eb-8430-1b684df7b324.png)